i18n/translations/en.json

```diff
@@ -369,6 +369,16 @@
 
 	"filter": "Filter",
 
+	"form.discard": "Discard changes",
+	"form.discard.confirm": "Do you really want to <strong>delete all unpublished changes</strong>?",
+	"form.locked": "This content is disabled for you as it is currently edited by another user",
+	"form.preview": "Preview changes",
+	"form.preview.draft": "Preview draft",
+	"form.publish": "Publish",
+	"form.published": "Published",
+	"form.save": "Save",
+	"form.saved": "Saved",
```

Comment:
`Thought` I'm always torn with translation variables. `save` and `saved`, `publish` and `published` are universally useful and we already have `save` and `saved` at least. Do we prefer more of those global translation strings or do we rather clarify the usage here by prefixing it with form? I see pros and cons in both ways.

Reply:
I can see the argument for save/saved/publish/published. I think the others are too specific to the context and we shouldn't try to use them as generics.

Reply:
I agree. Maybe we can create `publish` and `published` as generic ones in this pr, reuse the generic `save` and `saved` and move the rest to `form.` (as you already have)?